src/Motor.Extensions.Diagnostics.Tracing/TracingHostBuilderExtensions.cs

```diff
@@ -43,7 +43,7 @@ public static IMotorHostBuilder ConfigureJaegerTracing(this IMotorHostBuilder ho
 
                         builder
                             .AddAspNetCoreInstrumentation()
-                            .AddSource(typeof(TracingDelegatingMessageHandler<>).FullName!)
+                            .AddSource(openTelemetryOptions.Sources.ToArray())
```

Comment:
Do we want to include the motornet default source each time? Or should be over-ridable.

Reply:
can you think of any reason why it shouldn't be overridable?
In general I prefer to be as customizable as possible with sane defaults so that I'm able to change it if required but in most cases I don't need a lot of changes?

Reply:
People forget it to add the value again to source list. We can change it later, if some has this issue and in the example would could show how it works right now.